app/assets/stylesheets/petitions/_forms.scss

```diff
@@ -26,6 +26,10 @@ select.form-control:hover {
 .form-group {
   margin-bottom: $gutter;
   clear: both;
+
+  input[type=file] {
+    line-height: 1ex;
+  }
```

Comment:
I'm probably not the best person to answer that - https://github.com/alphagov/govuk_elements/tree/master/public/sass/elements/forms doesn't have any file specific input options. Without this, the 'file upload' button is mis-aligned with it's related text. @henryhadlow might be the best person to make a suggestion here?